frontend/src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx

```diff
@@ -414,7 +414,7 @@ const getOptionsFromLinkParams = (linkParams: LinkParams, jaegerInfo?: JaegerInf
 const mapStateToProps = (state: KialiAppState) => ({
   duration: durationSelector(state),
   updateTime: state.graph.updateTime,
-  jaegerInfo: state.jaegerState.info,
+  jaegerInfo: state.tracingState.info,
```

Comment:
If the state property is generic (tracingState), the variable should be renamed to tracingInfo